mysql/infoschema_test.go

```diff
@@ -43,28 +43,30 @@ func TestProcessInfoSchemaMYSQL(t *testing.T) {
 			rows: [][]driver.Value{
 				{"user"},
 				{"cart"},
-				{"test"}},
+				{"product"},
```

Comment:
Why are their changes to infoschema_test.go?
The focus of this PR is handling the foreign key case issue, and that is addressed by changes in mapping.go and toddl.go. I don't see why we'd have changes in infoschema_test.go. Can you clarify what these changes test that's relevant to this PR?
Is this left-over from a previous version of the PR?

Reply:
This change was affecting tests in infoschema_test.go, as we require for referenced tables to exist, there were couple of tests in infoschema_test.go which did not include referenced tables as it was not restricted earlier. so needed to add those referenced tables and referenced columns.

Reply:
Ah, of course --- thanks Pradip!